package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@timetac/js-client-library",
-  "version": "0.2.7",
+  "version": "0.3.0",
```

Comment:
set version to 0.3.0 because it has breaking changes
* deltaSync changed
* readRaw removed